File: 프로그래머스/unrated/181922. 수열과 구간 쿼리 4/수열과 구간 쿼리 4.py
def solution(arr, queries):
    answer = []
    for i in range(len(queries)):
        arr1 = []
        for j in range(queries[i][0], queries[i][1] + 1):
            if j % queries[i][2] == 0:
                arr[j]+=1
    answer = arr
    return answer

File: 프로그래머스/unrated/181922. 수열과 구간 쿼리 4/수열과 구간 쿼리 4.java
class Solution {
    public int[] solution(int[] arr, int[][] queries) {
        // int[] answer = new int[queries.length];
        int qNum = queries.length;
        for(int i = 0; i < qNum; i++){
            for(int j = queries[i][0]; j <= queries[i][1]; j ++){
                if(j % queries[i][2] == 0){
                    arr[j]++;
                }
            }
        }
        return arr;
    }
}